### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by alexax66**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+#: software-station.py:108 software-station.py:109
+msgid "Installed Software"
+msgstr "Установленные приложения"
+
+#: software-station.py:122
+msgid "Search Software"
+msgstr "Поиск приложений"
+
+#: software-station.py:129
```
Comment (ericbsd) [inline po/ru.po:44]: software-station does not have a `.py` are you sure it is working?

Comment (alexax66) [reply po/ru.po:44]: I renamed  software-station to software-station.py manaly for makin tarnslation. Then i used  msgmerge and msgfmt for bilding software-station.pot, ro.po and  zh_CN.po.

Comment (ericbsd) [reply po/ru.po:44]: Ok, so that might broke the previous work. The default is not software-station.py.

Comment (alexax66) [reply po/ru.po:44]: Снимок экрана в 2022-09-01 11-38-09

Comment (ericbsd) [reply po/ru.po:44]: good enough, I guess.

**Review by ericbsd — APPROVED** [review]
LGTM

**Comment by ericbsd** [comment]
Thanks

**Pull request merged**